server/core/src/main/scala/org.apache.daffodil.debugger.dap/DAPodil.scala

```diff
@@ -173,7 +173,7 @@ class DAPodil(
       case extract(Command.PAUSE, _)                                     => pause(request)
       case extract(Command.DISCONNECT, args: DisconnectArguments)        => disconnect(request, args)
       case extract(Command.EVALUATE, args: EvaluateArguments)            => eval(request, args)
-      case _                                                             => Logger[IO].warn(show"unhandled request $request") *> session.sendResponse(request.respondFailure())
```

Comment:
You probably want to keep the original formatting.

Reply:
The updated version of scalafmt says that is the proper formatting

Reply:
It also says to drive into a lake.  Not a big deal.  I'll allow it.